content/docs/integrating-with-other-libraries.md

```diff
@@ -100,17 +100,17 @@ componentWillUnmount() {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/qmqeQx?editors=0010)
+[**CodePen'de Deneyin**](https://codepen.io/gaearon/pen/qmqeQx?editors=0010)
 
-Note that React assigns no special meaning to the `this.el` field. It only works because we have previously assigned this field from a `ref` in the `render()` method:
+Dikkat edin ki, React `this.el` alanına özel bir anlam atamaz. O sadece, öncesinde bu alana `render()` metodundaki bir `ref`'den atama yapılmış olduğunda çalışır:
```

Comment:
To prevent the confusion, let's use `tahsis etmez` instead of `atamaz`.

Reply:
What about the second one in the second sentence: 
( .......`ref`'den **atama** yapılmış olduğunda çalışır.)